sale_report_hide_price/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Hide Price and Discount in Quotation Report',
+    'author': 'Serpent Consulting Services Pvt. Ltd.',
+    'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+    'category': 'Sales Management',
+    'website': 'https://www.serpentcs.com',
+    'version': '17.0.1.0.1',
+    'license': 'AGPL-3',
```

Comment:
@ShwetaSerpentCS please check
Odoo 17 Community Edition is licensed under LGPL version 3

Reply:
> @ShwetaSerpentCS please check Odoo 17 Community Edition is licensed under LGPL version 3

Please review updated code. @Trivedi-Vacha-SerpentCS 

Thank You.